### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#185] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-20 223929](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/1cf96046-3f39-4a9b-ad1b-03a3c7aeb05b)

**Comment by Elia-Belli** [comment]
> Analogo all'esercizio in classe dei clienti alle poste che usufruiscono del servizio verde o azzurro

Sappiamo che $X\sim Poisson(\lambda)$ è la v.a. che indica il numero di lanci $\Rightarrow \mathbb{P}(X=n)=e^{-\lambda }\cdot \frac{(\lambda)^n}{n!}$
Siano poi $T,C$ le v.a. che indicano rispettivamente il numero di teste e croci ottenute sugli $X$ lanci:

$$\mathbb{P}(T=t)=\sum^\infty_{n=t}\mathbb{P}(X=n)\mathbb{P}(T=t|X=n)=\sum^\infty_{n=t}e^{-\lambda }\cdot \frac{\lambda^n}{n!} \binom{n}{t}p^t(1-p)^{n-t}=\frac{e^{-\lambda}p^t}{t!}\sum^\infty_{n=t}\frac{\lambda^n}{(n-t)!}(1-p)^{n-t}=$$

$$=^{(h=n-t)}\frac{e^{-\lambda}p^t}{t!}\sum^\infty_{h=0}\frac{\lambda^{(h+t)}}{h!}(1-p)^h=\frac{e^{-\lambda}p^t}{t!}\cdot \lambda^t \cdot e^{\lambda (1-p)}= e^{(-\lambda p)}\cdot \frac{(\lambda p)^t}{t!}$$

Analogamente per $C$:

$$\mathbb{P}(C=c)=e^{(-\lambda (1-p))}\cdot \frac{(\lambda(1-p))^c}{c!}$$

**Comment by CiottoloMaggico** [comment, marked as answer]
## Soluzione confermata in classe
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/c7c90957-83d4-4d8a-8cc5-23d947c1c638)

**Comment by LeoCosta02** [comment]
![Screenshot (2)](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/7c7d88ba-be4c-4731-a34d-3657f48da27d)
@CiottoloMaggico come hai trasformato la serie in quell'esponenziale ?

**Comment by CiottoloMaggico** [comment]
Basta osservare che quella sommatoria coincide con il polinomio di Taylor di e^(c(1-p)) centrato in 0. Per intenderci:
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/0fae8535-c773-4c35-9313-337f3558d6ac)
**Nota che quel (e^x0)^(k) è la derivata k-esima di e^x0 e non una sua potenza**

**Comment by LeoCosta02** [comment]
@CiottoloMaggico grazie
